engines/dfc_provider/app/controllers/dfc_provider/api/v0/base_controller.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+# Abstract controller used to provide common logic to DFC APIs
+module DfcProvider
+  module Api
+    module V0
+      class BaseController < ::Api::V0::BaseController
+        # Skip the authorization check from the main OFN app
+        skip_authorization_check
+
+        before_action :check_authorization_with_access_token
+
+        respond_to :json
+
+        private
+
+        def check_authorization_with_access_token
+          return if access_token.present? && current_user.present?
+
+          unauthorized
```

Comment:
@Matt-Yorkley I have refactored the authorization check to one method more readable, and added a comment for the reason we are skipping the main check.